tests/l2/dsa/README.md

```diff
@@ -29,13 +29,13 @@ $ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-
 
 * Check the results.
 ``` 
-  $ oc get pods
+  $ oc get pods -n <namespace_name>
```

Comment:
should we directly point user the namespace name? like
$ oc get pods -n intel-dsa
Otherwise, user has to figure out the name.

Reply:
@uMartinXu thanks for the feedback, changes applied it